modules/liburing/2.5/patches/add_build_file.patch

```diff
@@ -0,0 +1,37 @@
+--- /dev/null
++++ a/BUILD.bazel
+@@ -0,0 +1,26 @@
++_COPTS = [
++    "-D_GNU_SOURCE",
++    "-D_LARGEFILE_SOURCE",
++    "-D_FILE_OFFSET_BITS=64",
++    "-DLIBURING_INTERNAL",
++    "-O3",
++    "-Wall",
```

Comment:
This is risky as different compilers might have different sets of warnings.

Reply:
This follows the upstream Makefile of the library. Anything that would fail here would by definition not be supported by upstream. Is that a reasonable reason to keep it? Otherwise I can drop it.

Reply:
Could you link to upstream? Differing in warning flags only sounds okay and is definitely better than random failures with other compilers.